### PR TITLE
Eleasah halsmer

### PR DESCRIPTION
**Pull request by ehalsmer**

### CODE REVIEW

**Comment by ehalsmer** [comment]
Added blog.html for stretch

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+</head>
+
+<body>
+<!-- navigation code here -->
+  <nav>
+    <h1>SpaceWalkers</h1>
+    <div class="nav-links">
+      <a href="index.html">Home</a>
+      <a href="#">About Us</a>
+      <a href="blog.html">Blog</a>
```
Comment (AnnaTownsend9916) [inline blog.html:23]: Great job, yet again you have soared beyond expectations of today

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: 'Russo One', sans-serif;
+  color: #23293B;
+}
+h2 {
+  font-size: 3.2rem;
```
Comment (AnnaTownsend9916) [inline css/index.css:149]: nice Job using rem

File: css/index.css [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+}
+.home .intro .header-img {
+  display: inline-block;
+  width: 100%;
+  height: auto;
+}
+.home .intro h2 {
+  padding: 15px;
+}
+@media (max-width: 500px) {
```
Comment (AnnaTownsend9916) [inline css/index.css:212]: great media queries

**Review by AnnaTownsend9916 — COMMENTED** [review]
Great job! You are fantastic and an amazing coder. Great work on doing stretch and nice Mixins 🥇 💯

**Pull request merged**